editor.planx.uk/src/@planx/components/List/Public/index.test.tsx

```diff
@@ -14,6 +15,8 @@ const mockProps: Props = {
   description: "Mock description",
 };
 
+jest.setTimeout(20_000);
```

Comment:
This test file is relatively slow, but passing well within this timeout locally. Failed on CI at the default 5000ms, and at 10,000ms.

![image](https://github.com/theopensystemslab/planx-new/assets/20502206/cc614cb8-9506-4109-b6ca-95ce11d1673f)